lms/envs/common.py

```diff
@@ -2295,3 +2295,6 @@
 PROFILE_IMAGE_SECRET_KEY = 'placeholder secret key'
 PROFILE_IMAGE_MAX_BYTES = 1024 * 1024
 PROFILE_IMAGE_MIN_BYTES = 100
+
+# This is to check the domain in case of preview.
+PREVIEW_DOMAIN = 'preview'
```

Comment:
@muzaffaryousaf 

Thank you for your help!

Did you try to change some course parts before click preview button??
"preview" means, we can preview course in "draft stage", not publish stage.
In my case, If I use some name other than "preview" then preview button show me "publish stage".
This is why I said not working.

Sorry for my poor English.